### PR TITLE
Introduces copy to clipboard button

### PR DESCRIPTION
**Pull request by KaiVolland**
All credits go to @dnlkoch. <3

:hankey: with :heart: by terrestris

### CODE REVIEW

File: src/util/CopyClipboard.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Ext.define('BasiGX.util.CopyClipboard', {
```
Comment (marcjansen) [inline src/util/CopyClipboard.js]: No API docs :disappointed:

File: src/util/CopyClipboard.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    statics: {
+
+        /**
+         *
+         */
+        copyToClipboardSupported: document.queryCommandSupported('copy'),
```
Comment (marcjansen) [inline src/util/CopyClipboard.js]: Does IE or Edge have `queryCommandSupported`?

Comment (dnlkoch) [inline src/util/CopyClipboard.js]: Yes, see [here](https://developer.mozilla.org/en-US/docs/Web/API/Document/queryCommandSupported#Browser_compatibility).

File: src/util/CopyClipboard.js (hunk 1 above)
Comment (dnlkoch) [inline src/util/CopyClipboard.js]: You're so right. Added in 7ac3c0f4.

**Pull request merged**